README.md

```diff
@@ -9,3 +9,8 @@
 - `git push -u origin main` / `git push`
 - `git remote -v`
 - `git remote add origin <ssh link>`
+
+## Git Branches & PRs
+
+- `git switch -c <branch-name>`
+- `git switch <branch-name`
```

Comment:
Well done, thank you. You could also add `git switch -` to switch to the branch where we were before. :-)